price-feeder/oracle/client/client.go

```diff
@@ -125,7 +139,7 @@ func (oc OracleClient) BroadcastTx(nextBlockHeight, timeoutHeight int64, msgs ..
 
 	// re-try voting until timeout
 	for lastCheckHeight < maxBlockHeight {
-		latestBlockHeight, err := rpcClient.GetChainHeight(clientCtx)
+		latestBlockHeight, err := oc.ChainHeight.GetChainHeight()
 		if err != nil {
 			return err
```

Comment:
What happens here? E.g., in the case of the chain stopping, chainheight returns an error, and the loop continues? 

Just want to make sure that in the event of a chain halt, validators would not have to restart all their price feeders... they'll likely be hands-off and we wouldn't want to have to worry about everyone getting jailed

Reply:
Well, if the chain stops it will only return an error if the unsubscribe of the EventNewBlockHeader returns an error, On the old rpcClient.GetChainHeight what would need to happen to return an error?

Reply:
RPC would have to be unavailable - sounds like this loop wouldn't stop though 👍

Reply:
that's why I proposed to use `maxTries` instead. @RafilxTenfen can you add a task for that?